ui/components/multichain/edit-networks-modal/edit-networks-modal.js

```diff
@@ -47,11 +64,11 @@ export const EditNetworksModal = ({ onClose }) => {
         </Box>
         {nonTestNetworks.map((network) => (
           <NetworkListItem
-            name={network.nickname}
-            iconSrc={network?.rpcPrefs?.imageUrl}
-            key={network.id}
+            name={network.name}
+            iconSrc={CHAIN_ID_TO_NETWORK_IMAGE_URL_MAP[network.chainId]}
```

Comment:
One thing you'll notice throughout, is that the network image is now selected directly via constants.  The network controller has never supported an image url in its state, but at some point it leaked in.  After `providerConfig` was removed in https://github.com/MetaMask/metamask-extension/pull/26150, it will no longer incorrectly be in state anyway.  To my knowledge, we've never supported users providing custom network images, so I think the source was always just the constants to begin with.